src/main/apiClient.ts

```diff
@@ -123,6 +133,8 @@ export default class ApiClient {
     this.authorizeUrl = `https://${this.config.domain}/oauth/authorize`
     this.tokenUrl = `https://${this.config.domain}/oauth/token`
     this.popupRelayUrl = `https://${this.config.domain}/popup/relay`
+    this.startPhoneNumberRegistrationUrl = `https://${this.baseUrl}/mfa/credentials/register/phone-number/start`
+    this.verifyPhoneNumberRegistrationUrl = `https://${this.baseUrl}/mfa/credentials/register/phone-number/verify`
```

Comment:
We can do without those: most client methods hardcode the URL in their impl. The above (`authorizeUrl` etc) are defined because they're used > 1 times.